docs/src/templates/includes/components/core/je-restaurant-listing.hbs

```diff
@@ -1,27 +1,30 @@
-<section class="c-listing-item c-card" itemscope="" itemtype="http://schema.org/Restaurant">
-    <a href="#" class="c-mediaElement c-listing-item-link" itemscope="" itemtype="http://schema.org/Restaurant" title="View the Menu for Fozzie Kebab">
-        <div class="c-listing-item-img c-mediaElement-img">
-            <img src="{{ baseUrl }}/assets/img/examples/restaurant-logo.gif" alt="Fozzie Kebab" itemprop="image">
+<section class="c-listing-item" itemscope="" itemtype="http://schema.org/Restaurant" data-restaurant-id="123" data-test-id="123">
+    <a href="#" class="c-listing-item-link u-clearfix" itemscope="" itemtype="http://schema.org/Restaurant"
+       title="View the Menu for Fozzie Kebab">
+        <div class="c-listing-item-img">
+            <img class="js-lazy c-listing-item-imgItem" data-ft="restaurantDetailsLogo" src="{{ baseUrl }}/assets/img/examples/restaurant-logo.gif" alt="Fozzie Kebab" />
+            <noscript>img class="c-restaurant__logo js-lazy" data-ft="restaurantDetailsLogo" src="{{ baseUrl }}/assets/img/examples/restaurant-logo.gif" alt="Fozzie Kebab" itemprop="image";</noscript>
         </div>
-        <div class="c-listing-item-info c-mediaElement-content g g--stack--midWide">
-            <div class="g-col g-span7 g-span8--wide c-listing-item-col">
-                <h3 class="c-listing-item-title" itemprop="name">Fozzie Kebab</h3>
-                <div itemprop="aggregateRating" itemscope="" itemtype="http://schema.org/AggregateRating">
-                    <p class="c-listing-item-text">Rating 5.5 out of 6</p>
-                    ( 150 )
-                    <meta itemprop="ratingValue" content="5.5">
-                    <meta itemprop="itemreviewed" content="">
-                    <meta itemprop="ratingCount" content="150">
-                    <meta itemprop="bestRating" content="6">
-                    <meta itemprop="worstRating" content="0">
-                </div>
-                <p class="c-listing-item-text" itemprop="servesCuisine">Cuisine Type</p>
-                <p class="c-listing-item-text u-showAboveMid" itemprop="address" itemscope="" itemtype="http://schema.org/PostalAddress">123 Fake Street, Bristol</p>
-            </div>
-            <div class="g-col g-span5 g-span4--wide g--alignSelfBottom c-listing-item-col u-showAboveMid">
-                <p class="c-listing-item-text">Delivery fee: £1</p>
-                <p class="c-listing-item-text">Spend £10 for delivery</p>
+        <div class="c-listing-item-meta">
+            <span class="c-listing-item-label c-listing-item-flag">20% off</span>
```

Comment:
More of an open question for opinions – this doesn't currently have any semantic meaning as it's just a `span` inside a `div` – would it be better for it to use a `p` tag for better accessibility with screen readers etc?

/cc @simonsandersje @DamianMullins

Reply:
bump 😁

Reply:
Sorry, I completely missed this comment!

Sounds reasonable to me, it would be good to test this out with a screen reader to see what effect changing to a paragraph would have?